ground/src/main/java/com/google/android/ground/persistence/local/room/converter/GeometryWrapperTypeConverter.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.persistence.local.room.converter
+
+import androidx.room.TypeConverter
+import com.google.android.ground.persistence.local.room.entity.GeometryWrapper
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.json.JSONException
+import timber.log.Timber
+
+object GeometryWrapperTypeConverter {
+
+  @TypeConverter
+  fun toString(geometryWrapper: GeometryWrapper?): String? =
+    geometryWrapper?.getGeometry()?.let { Json.encodeToString(it) }
+
+  @TypeConverter
+  fun fromString(jsonString: String?): GeometryWrapper? =
```

Comment:
Any reason we can't/shouldn't use CBOR here instead?
https://kotlinlang.org/docs/serialization.html#formats

Otherwise we're back in the same b(l)oat as before with GeoJSON..

Reply:
I don't see a reason why not.